Corona-Warn-App/src/main/java/de/rki/coronawarnapp/covidcertificate/vaccination/ui/details/VaccinationDetailsFragment.kt

```diff
@@ -190,4 +202,15 @@ class VaccinationDetailsFragment : Fragment(R.layout.fragment_vaccination_detail
             }
         }.show()
     }
+
+    private fun showExportErrorDialog() {
```

Comment:
Can you extract the dialog implementation? You are using exactly the same code 3 times